crates/interledger-store-redis/src/store.rs

```diff
@@ -83,15 +84,19 @@ local to_amount = tonumber(ARGV[2])
 local balance = redis.call('HINCRBY', to_account, 'balance', to_amount)
 local prepaid_amount, settle_threshold, settle_to = unpack(redis.call('HMGET', to_account, 'prepaid_amount', 'settle_threshold', 'settle_to'))
 
--- Check if we should send a settlement for this account
+-- The logic for trigerring settlement is as follows:
+--  1. settle_threshold must be non-nil (if it's nil, then settlement was perhaps disabled on the account).
+--  2. clearing_balance must be greater than settle_threshold (this is the core of the 'should I settle logic')
```

Comment:
The variable is called `balance` here

```suggestion
--  2. balance must be greater than settle_threshold (this is the core of the 'should I settle logic')
```